python/paddle/fluid/contrib/slim/tests/QAT_mkldnn_int8_readme.md

```diff
@@ -116,9 +116,14 @@ OMP_NUM_THREADS=28 FLAGS_use_mkldnn=true python python/paddle/fluid/contrib/slim
 OMP_NUM_THREADS=28 FLAGS_use_mkldnn=true python python/paddle/fluid/contrib/slim/tests/qat_int8_comparison.py --qat_model=/PATH/TO/DOWNLOAD/MODEL/${MODEL_NAME} --infer_data=~/.cache/paddle/dataset/int8/download/int8_full_val.bin --batch_size=50 --batch_num=1000 --acc_diff_threshold=0.01 --qat2
 ```
 
-- Performance benchmark
+* Performance benchmark
 
 ```bash
-OMP_NUM_THREADS=1 FLAGS_use_mkldnn=true python python/paddle/fluid/contrib/slim/tests/qat_int8_comparison.py --qat_model=/PATH/TO/DOWNLOAD/MODEL/${MODEL_NAME} --infer_data=~/.cache/paddle/dataset/int8/download/int8_full_val.bin --batch_size=1 --batch_num=1000 --acc_diff_threshold=0.01 --qat2
+# 1. Save QAT_INT8 model
+python  /home/danqing/repo/Paddle/python/paddle/fluid/contrib/slim/tests/qat_save_model.py --qat_model_path /PATH/TO/DOWNLOAD/MODEL/${QAT2_MODEL_NAME} --qat_int8_model_path /PATH/TO/${QAT2_MODEL_NAME}_qat_int8
```

Comment:
`/home/danqing/repo/Paddle/python/paddle/fluid/contrib/slim/tests/qat_save_model.py` --> `PATH/To/Paddle/python/paddle/fluid/contrib/slim/tests/qat_save_model.py`